sematic/ui/packages/common/src/pages/RunSearch/SearchFilters.tsx

```diff
@@ -80,8 +82,20 @@ const SearchFilters = (props: SearchFiltersProps) => {
         onFiltersChanged({...allFilters.current});
     }, [onFiltersChanged]);
 
+    const [submitedDefaults, setSubmitedDefaults] = useState(false);
+    useEffect(() => {
+        if(submitedDefaults) {
+            return;
+        }
+        if(!!defaultSearchString) {
+            allFilters.current[FilterType.SEARCH] = [defaultSearchString];
+        }
+        applyFilters();
```

Comment:
This might cause the search filter being applied twice in the initial rendering and make two server requests. Can this behavior be observed on your side?

Reply:
yes, when I deployed it to my namespace (where there are more runs, and search latency is likely slower) I noticed two renders of the search results on load. Will adjust to make the search index page load the default filter from the url and pass that down to both the list display and the search filter display.